compiler/luci/profile/include/luci/Profile/CircleNodeOrigin.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PROFILE_CIRCLE_NODE_ORIGIN_H__
+#define __LUCI_PROFILE_CIRCLE_NODE_ORIGIN_H__
+
+#include "CircleOriginData.h"
+
+#include <luci/IR/CircleNode.h>
+
+#include <vector>
+
+namespace luci
+{
+
+using CircleNodeOrigin = std::vector<CircleOriginData>;
+
+bool has_node_origin(const luci::CircleNode *circle_node);
+
+void add_node_origin(luci::CircleNode *circle_node, const CircleOriginData &origin);
+
+const CircleNodeOrigin &get_node_origin(const luci::CircleNode *circle_node);
```

Comment:
(just thougts)  we are handling for `CircleNode` and there is an argument `luci::CircleNode *`
so for shorter name, we may use `has_origin`, `add_origin`, `get_origin`.
and as `get_origin` returns a vector of `CircleOriginData`, we may use `get_origins` to denote it returns multiple of origins.

If we use `CircleOrigin` instead of `CircleOriginData` then we may use `CircleOrigins = std::vector<CircleOrigin>;`

Reply:
- `has_origin`, `add_origin`, `get_origin` : LGTM
- `has_origins`, `add_origins`, `get_origins` : LGTM. But for this, renaming file to `CircleNodeOrigins` may be needed.

Reply:
`has_origin` vs `has_origins` :  seems similar, to ask if there is any origin or origins
`add_origin` vs `add_origins` : `add_origin` will add one item. `add_origins` will add multiple items
`get_origin` vs `get_origins` : `get_origin(luci::CircleNode*, int index)` would make sense but not sure this is needed. so for the purpose of origin, we may only need `get_origins`